keepassxc-browser/background/page.js

```diff
@@ -153,6 +158,23 @@ page.initOpenedTabs = async function() {
     }
 };
 
+page.initSitePreferences = async function() {
+    if (!page.settings) {
+        return;
+    }
```

Comment:
Since I am initializing, shouldn't it create the `page.settings` if it doesn't already exist? This seems like we can be left with a totally uninitialized state or functions are being called out of order.

Reply:
Gotta double check this.

Reply:
The settings are created before calling this function. So it's just a check, possibly not necessary.